core/src/types/error.rs

```diff
@@ -227,8 +227,9 @@ pub struct Error {
     status: ErrorStatus,
     operation: &'static str,
     context: Vec<(&'static str, String)>,
+
     source: Option<anyhow::Error>,
-    backtrace: Backtrace,
+    backtrace: Option<Box<Backtrace>>,
```

Comment:
What about boxing everything except `kind`?

Reply:
It's a tradeoff between "reducing the size from 88 to even smaller" and "always having a box allocation." I prefer to avoid the box allocation.